client/web/src/repo/tree/TreePagePanels.module.scss

```
@@ -2,7 +2,7 @@
     max-height: 30rem;
     overflow: hidden;
     margin-top: 1rem;
-    padding: 0 0.25rem 0 0;
+    padding: 0 0.2rem 0 0.2rem;
```

rems are translated to pixels 1:16 (by default; user can change this in browser settings), so 0.2rem does not translate to an integer pixel value. I recommend using 0.25 or 0.125 here.

---

This should probably do it:

```suggestion
    padding: 0 0.25rem;
```